src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -192,8 +192,8 @@ var HTMLDOMPropertyConfig = {
      */
     // autoCapitalize and autoCorrect are supported in Mobile Safari for
     // keyboard hints.
-    autoCapitalize: null,
-    autoCorrect: null,
+    autoCapitalize: MUST_USE_ATTRIBUTE,
+    autoCorrect: MUST_USE_ATTRIBUTE,
```

Comment:
Note: now that we're not using properties, can you remove the entries in the `DOMPropertyNames` mapping below. We do not need to add them to the `DOMAttributeNames` mapping.